core/Cargo.toml

```diff
@@ -64,6 +66,12 @@ wasmtime = "0.36.0"
 
 # transitive dependencies
 anyhow = ">= 1.0"
+dyn-clone = "1.0.5"
+erased-serde = "0.3.20"
```

Comment:
I don't like seeing introduced two completely new dependencies. Could you explain why it's required? It seems to me that now we have two approaches in the codebase for serialization of trait objects. The one where we make boxed enums with variants of all types implementing a trait like `IdentifiableBox` and the 2nd approach introduced here

Reply:
Yeah, you are right. This was my first attempt which failed at deserialization step. But `Clone` and `Serialize` for trait objects like `IsAllowedBoxed` should work fine.

Although it seems like we don't need it, cause it the future there will be an enum for Genesis permissions (de-)serialization.

I'll remove this things then